### PR TITLE
Update models.py

### PR DESCRIPTION
**Pull request by allenazamoras**
I updated models.py for both livestream and UserProfile applications
I also updated views.py, serializers.py, and urls.py in livestream to accommodate for the changes in the model

### CODE REVIEW

File: livestream/views.py [hunk 1]
```diff
@@ -1 +1 @@
...
#             # VALUE ERROR SOLVED
#             ret = ClientTokenSerializer(new_token).data

#         return Response(ret)


class IndexView(generic.ListView):
    template_name = 'livestream/index.html'
    context_object_name = 'session_list'
    queryset = Appeal.objects.all()
```
Comment (aldnav) [inline livestream/views.py]: @allenazamoras I don't understand what's going on in here.

File: aLive/settings.py [hunk 2]
```diff
@@ -36,14 +36,17 @@
...
+
+CORS_ORIGIN_ALLOW_ALL = True
 
 # Application definition
 
 INSTALLED_APPS = [
     'rest_framework',
     'rest_framework.authtoken',
     'corsheaders',
+    'UserProfile.apps.UserprofileConfig',
```
Comment (aldnav) [inline aLive/settings.py]: @allenazamoras can you refactor your app and update the name of `Userprofile` app into `userprofile` all lowercase?

File: UserProfile/models.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    first_name. last_name, email, is_staff, is_active, date_joined
+    """
+
+    reputation = models.PositiveIntegerField(default=0)
+
+    def __str__(self):
+        return self.username
+
+
+@receiver(post_save, sender=User)
```
Comment (aldnav) [inline UserProfile/models.py]: @allenazamoras isolate this signal receiver into it's own module say `signal_listeners` or `signal_receivers`. Then, update [apps.py](https://github.com/ephraimeg/alive-project/blob/3636cb0c4458151bb223da60dcc9c77cc9e026ef/UserProfile/apps.py) to include,

```python
    # ...
    name = 'userprofile'
    import signal_listeners
```

This way, you separate the model and the signal concern and also make Django load your signal listeners.

File: livestream/views.py (hunk 1 above)
Comment (allenazamoras) [reply livestream/views.py]: I don't don't why it's like that sir, but I think it has something to do with the views.py in the development branch that I cloned. It also appears like that when I inspect the view.py in livestream

File: UserProfile/models.py (hunk 3 above)
Comment (allenazamoras) [reply UserProfile/models.py]: should i do anything else aside from importing signal_receivers? I'm getting a 
`ImportError: No module named 'signal_receivers'` when i try to run the server

**Comment by aldnav** [comment]
Needs a follow-up PR to fix this loading of [signal listener](https://github.com/ephraimeg/alive-project/blob/development/userprofile/apps.py#L6).

**Pull request merged**